tests/test_externe_referenz.py

```diff
@@ -61,3 +63,14 @@ def test_geschaeftspartner_with_no_externe_referenz(self) -> None:
         deserialized_gp: Geschaeftspartner = Geschaeftspartner.parse_raw(gp_json)
 
         assert deserialized_gp.externe_referenzen == []
+
+    def test_extension_data(self) -> None:
+        """
+        tests the behaviour of the json extension data (`extra="allow"`)
+        """
+        er = ExterneReferenz(ex_ref_name="foo.bar", ex_ref_wert="12345")
+        er_json: Dict[str, Any] = er.dict()
+        er_json["additional_key"] = "additional_value"
+        deserialized_er: ExterneReferenz = ExterneReferenz.parse_obj(er_json)
+        assert isinstance(deserialized_er, ExterneReferenz)
+        assert deserialized_er.additional_key == "additional_value"  # type:ignore[attr-defined]
```

Comment:
nur für mein Verständnis, das hätte auch ohne das `extra = "allow"` geklappt oder?

Reply:
Nein, genau dafür is extra="allow" da. Siehe
>      def test_extension_data(self) -> None:
>        """
>        tests the behaviour of the json extension data (`extra="allow"`)
>        """
>        er = ExterneReferenz(ex_ref_name="foo.bar", ex_ref_wert="12345")
>        er_json: Dict[str, Any] = er.dict()
>        er_json["additional_key"] = "additional_value"
>        deserialized_er: ExterneReferenz = ExterneReferenz.parse_obj(er_json)
>        assert isinstance(deserialized_er, ExterneReferenz)
>       assert deserialized_er.additional_key == "additional_value"  # type:ignore[attr-defined]
> E       AttributeError: 'ExterneReferenz' object has no attribute 'additional_key'

https://github.com/bo4e/BO4E-python/pull/441